breakable_toy/docker-compose.yml

```diff
@@ -23,3 +23,14 @@ services:
     image: redis:3.2
     expose:
     - "6379"
+  leaflet:
```

Comment:
- Blank lines are fine in YAML, and putting a blank line between service definitions helps with readability.

- It's generally good practice to match the name of a service and the directory in which you put that service's files.  Removes any ambiguity and means fewer identifiers to keep track of.

- The name can be the name of the actual program (e.g. "windshaft", "nginx", "postgres") or it can be the role it fills ("tile-server", "web", "database") or something else, if it seems like the most apt indentifier.  Some projects might have two different web servers and name them by function (e.g. "public-site" and "management-site").  In this case "webpage" or "web" seem reasonable to me.  "Leaflet" as the name of this service seems a little indirect to me, since what it's actually doing is running an nginx container that serves a wab page that loads Leaflet from a CDN and uses it.

- Notwithstanding the above, actually this demo could be done with no new service at all--we could skip having the HTML file served by the nginx container and just open the HTML in the browser directly:
![image](https://user-images.githubusercontent.com/6598836/41249932-77c2924e-6d83-11e8-8578-d4e0a50962ac.png)

The nginx container also works fine, and obviously if the project were going to deploy with an nginx service serving static content then it would make sense to use it for development as well.  I just wanted to point out that for a static demo like this, you can get away with very little infrastructure.

Reply:
That's a good point about the nginx service, my original thought was to try and compartmentalize the system by only exposing Windshaft to the nginx server rather than to a localhost port. When I realized that that isn't how that works (since you have to load the tiles from client-side JS anyway) I didn't think to get rid of the service.